### PR TITLE
Make sure transfer names can begin with dash

### PR DESCRIPTION
**Pull request by sevein**
This is connected to #1129.

### CODE REVIEW

File: src/dashboard/src/main/migrations/0053_quote_args.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    tasks = (
+        # upload-archivesspace.py
+        (
+            '10a0f352-aeb7-4c13-8e9e-e81bda9bca29',
+            u'--host="%host%" --port="%port%" --user="%user%"'
+            u' --passwd="%passwd%" --dip_location="%SIPDirectory%"'
+            u' --dip_name="%SIPName%" --dip_uuid="%SIPUUID%"'
+            u' --restrictions="%restrictions%" --object_type="%object_type%"'
+            u' --xlink_actuate="%xlink_actuate%" --xlink_show="%xlink_show%"'
+            u' --use_statement "%use_statement%" --uri_prefix "%uri_prefix%"'
```
Comment (jrwdunham) [inline src/dashboard/src/main/migrations/0053_quote_args.py:19]: Why are you changing some of the optional arguments of the form `--x n` to the form `--x=n` but not others? E.g., `--use_statement "%use_statement%"` lacks the `=`. Is there a reason?

File: src/dashboard/src/main/migrations/0053_quote_args.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def data_migration_put_quotes_in_compress_aip_args(apps, schema_editor):
```
Comment (jrwdunham) [inline src/dashboard/src/main/migrations/0053_quote_args.py:7]: Does the function name here really reflect what it's doing? Relatedly, why are you targeting these and only these 6 `StandardTaskConfig`s? They seem to all contain values for `SIPName`. But there are 18 others:

    >>> len(models.StandardTaskConfig.objects.filter(arguments__contains='SIPName'))
    24

Why do the others not need mods to their arguments value also?

Maybe a docstring in this migration func would be helpful.

**Review by jrwdunham — COMMENTED** [review]
I have some questions ...

**Comment by sevein** [comment]
Hey @jrwdunham, thanks for your comments. I think you're right, this commit may need some more work as it doesn't seem to cover all the cases. Thanks.

**Pull request closed without merging**